scripts/init_sms_bridge.py

```
@@ -0,0 +1,149 @@
+    
+    if len(password) < 8:
+        print("! Password must be at least 8 characters")
+        sys.exit(1)
+    
+    create_initial_admin(username, password, email)
+    
+    # Create default settings
+    create_default_settings(created_by=username)
+    
+    print("\n✓ Setup complete!")
+    print(f"\nStart the server with:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)